docs/user/quick-start.md

```diff
@@ -0,0 +1,80 @@
+# Quick Start
+
+This Quick Start guide will cover:
+
+- Using kyma cli to provision control plane cluster and deploy Kyma Lifecycle Manager
+- Deploy module template
+- Using kyma cli manage modules
+
+## Prerequisites
+To use Kyma Lifecycle Manager for local setup, you need to have the following prerequisites:
+
+- [k3d](https://k3d.io/)
+- [istioctl](https://istio.io/latest/docs/setup/install/istioctl/)
+- [Kyma CLI](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/01-install-kyma-CLI)
+
+## Provisioning a Control Plane Cluster with required resources
+
+You can provision a local k3d cluster as a control plane using the following command, by default, a cluster named `k3d-kyma` will be created. 
+```
+kyma provision k3d
+```
+Since the services deployed in the control plane are managed by Istio, you need to install Istio into the local k3d cluster using the following command:
+```
+istioctl install -y
+```
+Lifecycle Manager also exposes metrics that are collected by Prometheus Operator in the control plane to provide better observability. To simplify the local setup, you only need to deploy the ServiceMonitor CRD using the following command:
+```
+kubectl apply -f hack/servicemonitors.yaml
```

Comment:
this hack file is not necessary, I think we could just use a URL for the servicemonitors crd

Reply:
unfortunately, there is no existing servicemonitors crd exists, current [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator/) will generate all resources as a bundle.yaml to create with. Maybe search online can find this archived [crd](https://github.com/helm/charts/blob/master/stable/prometheus-operator/crds/crd-servicemonitor.yaml), but that's already outdated, still use `apiVersion: apiextensions.k8s.io/v1beta1`

Reply:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/prometheus-community/helm-charts/main/charts/kube-prometheus-stack/crds/crd-servicemonitors.yaml
```
here you go :)